definitions/tools/bam_to_bigwig.cwl

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: 'cgpBigWig Converting BAM to BigWig'
+baseCommand: ["bam2bw", "-F", "1024", "-o", "MarkedSorted.bam.bw"]
```

Comment:
It'd be nice to have the output filename derived from the input filename instead of hardcoded.  (This probably requires moving the command-line arguments to the `arguments:` section instead of the `baseCommand:` section.)